src/com/hotmail/kalebmarc/textfighter/main/Saves.java

```diff
@@ -315,214 +225,224 @@ public static boolean load() {
+		set("Achievements.Aww_You_Care_About_Me", Achievements.awwYouCareAboutMe);
+		set("Achievements.Slayer", Achievements.slayer);
+		set("Achievements.Nobodys_Perfect", Achievements.nobodysPerfect);
+		set("Achievements.Making_Money", Achievements.makingMoney);
+		set("Achievements.Gambling_Addiction", Achievements.gamblingAddiction);
+		set("Achievements.Unnatural_Luck", Achievements.unnaturalLuck);
+		set("Achievements.Level_2_Fighter", Achievements.level2Fighter);
+		set("Achievements.Level_3_Fighter", Achievements.level3Fighter);
+		set("Achievements.Level_4_Fighter", Achievements.level4Fighter);
+		set("Achievements.Level_5_Fighter", Achievements.level5Fighter);
+		set("Achievements.Level_6_Fighter", Achievements.level6Fighter);
+		set("Achievements.Level_7_Fighter", Achievements.level7Fighter);
+		set("Achievements.Level_8_Fighter", Achievements.level8Fighter);
+		set("Achievements.Level_9_Fighter", Achievements.level9Fighter);
+		set("Achievements.Level_10_Fighter", Achievements.level10Fighter);
+		set("Achievements.Honest_Player", Achievements.honestPlayer);
+
+		//Other Stuff
+		set("Settings.About_Viewed", About.viewed());
+		set("Stats.Times_Cheated", Stats.timesCheated);
+		set("Stats.Times_Quit", Stats.timesQuit);
+		set("Stats.Items_Crafted", Stats.timesCheated);
+		set("Stats.Games_Played.Dice", Stats.diceGamesPlayed);
+		set("Stats.Games_Played.Slots", Stats.slotGamesPlayed);
 	}
 
-	public static boolean convert() {
-		Ui.cls();
-		Ui.println("------------------------------------");
-		Ui.println("WARNING- Converting a save file may");
-		Ui.println("result in a corrupt save.");
-		Ui.println("It's recommended that you make a");
-		Ui.println("backup of your current save file(s)");
-		Ui.println("before you continue.");
-		Ui.println("------------------------------------");
-		Ui.println("1) Exit");
-		Ui.println("2) Continue");
-
-		switch(Ui.getValidInt()){
-			case 1:
-				return false;
-			case 2:
-				break;
-			default:
-				return false;
-		}
+	// TODO: be more consistent with prompts across game
+	/**
+	 * Load a save game and set values to various variables
+	 *
+	 * @return true if successfully loaded
+	 */
+	public static boolean load() {
+		path = Saves.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+		path = path.replaceAll("%20", " ").substring(0, path.lastIndexOf('/'));
 
-		try {
-			File file = new File(path);
+		File dir = new File(path + "/");
+		System.out.println(dir);
 
-			if (!file.exists()) {
-				Ui.println("File not found. Please put an \"_\" before your username in the save file.");
-				System.exit(0); //TODO shouldn't just exit like this.... Go back to main menu
-			}
+		File[] saveList = dir.listFiles((dir1, name) -> name.endsWith(".TFsave"));
+
+		cls();
+		println("------------------------------");
+		println("Choose a save game...");
+		println("------------------------------");
+		for (int i = 0; i < saveList.length; i++)
```

Comment:
Its possible that saveList is empty, and theres no way to exit the blank "choose save game" menu

Reply:
Added a temporary menu option for exiting as option 0. In the future, I would like to implement pagination and the final option on each page would be to exit. I believe making all menus consistent would be a significant improvement as well.